backend/payment-service/src/main/kotlin/com/ticketqueue/payment/service/PaymentService.kt

```
@@ -0,0 +1,110 @@
+        }
+
+        try {
+            portoneClient.preRegisterPayment(
+                paymentId = paymentKey,
+                request = PortonePreRegisterRequest(
+                    storeId = storeId,
+                    totalAmount = request.amount.longValueExact()
+                ),
+                token = portoneTokenService.getAccessToken()
+            )
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)